generators.js

```diff
@@ -1244,6 +1244,21 @@ function noCloud() {
 	}
 	endShape(CLOSE);
 
+
+register(straubCloud, "Straub Cloud", "edwin.straub")
+
+function CloudyCloud()
+{
+	//I'm really sorry about this one xD
+	translate(width/2,height/2);
+	noStroke();
+	for(let i = 0; i < 30; i++)
+		ellipse(random(-130,130),random(-15,25),random(50,200));
+  
+  return [-130,-50,130*2,50*2];
+}
+register(CloudyCloud, "Radnomly Generated Cloud", "this.Zohir")
+=======
```

Comment:
Looks like this is left over from the merge resolution. Is that something you can fix easily? If not I'll fix it whilst merging!

Reply:
I don't understand, the return statement is mine

Reply:
Sorry, this wasn't very clear - I was commenting baout the line 1261: ======= - it's a remainder from the merge resolution you did. Don't worry, I'll fix it and merge :)